lib/scrabble_word.rb

```diff
@@ -0,0 +1,27 @@
+#Define a class named ScrabbleWord with an initialize method that takes one argument, word, and stores it in an instance variable @word.
+class ScrabbleWord
+  def initialize(word)
+    #Write a "getter" for this instance variable @word that returns the word as a lowercase string.
+    @word = word
```

Comment:
this is not a getter method for `word`, this creates the initial value for `@word`. In order to return the value for `@word`, you would need to write a getter, i.e.:

```ruby
def word
 @word # returns the value of @word
end
```